richtext/plugins/org.polarsys.kitalpha.richtext.common/src/org/polarsys/kitalpha/richtext/common/impl/AbstractMDERichTextWidget.java

```diff
@@ -14,12 +14,9 @@
 import java.beans.PropertyChangeListener;
```

Comment:
Space and tabulation problem?

Reply:
Yes, the file previously had empty lines with tabs.
Corrected indentation replaces spaces with tabs.